handler/rfc7523/handler_test.go

```diff
@@ -773,9 +773,9 @@ func (s *AuthorizeJWTGrantRequestHandlerTestSuite) createStandardClaim() jwt.Cla
 		Issuer:    "trusted_issuer",
 		Subject:   "some_ro",
 		Audience:  jwt.Audience{"https://www.example.com/token", "leela", "fry"},
-		Expiry:    jwt.NewNumericDate(time.Now().AddDate(0, 0, 23)),
+		Expiry:    jwt.NewNumericDate(time.Now().UTC().AddDate(0, 0, 23)),
 		NotBefore: nil,
-		IssuedAt:  jwt.NewNumericDate(time.Now().AddDate(0, 0, -7)),
+		IssuedAt:  jwt.NewNumericDate(time.Now().UTC().AddDate(0, 0, -7)),
```

Comment:
Had to change this because tests were failing locally for me

Reply:
I was a bit concerned by this one, as I also could reproduce it. Took me quite some time to understand it, but I finally got it:
This test uses the maximum configured JWT duration (720h = 30d). The problem is now with daylight saving, as `AddDate` respects that in an unexpected way:
https://go.dev/play/p/tqJSlsabYmr?v=

Reply:
Ooh interesting, should I remove this then and just switch `AddDate` with `Add`?

Reply:
I'd say either is fine, as this is just a test.